### ISSUE TITLE
Try catch not working

### ISSUE THREAD

**Issue by Noobware1** [ethanblake4/dart_eval#134]
v0.7 trying to do
```
Datetime? main() {
  final a = tryCatch();
  return a;
 }

DateTime? tryCatch() {
  try {
    return DateTime.parse('please fail');
  } catch (e) {
    return null;
  }
}
```

getting this error 
```
Unhandled exception:
dart_eval runtime exception: FormatException: Invalid date format
please fail
#0      DateTime.parse (dart:core/date_time.dart:358:7)
#1      $DateTime.$parse (package:dart_eval/src/eval/shared/stdlib/core/date_time.dart:374:36)
#2      InvokeExternal.run (package:dart_eval/src/eval/runtime/ops/bridge.dart:106:44)
at tryCatch()
at main()
```

**Comment by ethanblake4** [comment]
Pushed fix for this to v0.7.0 branch

**Comment by ethanblake4** [comment]
Fixed in v0.7.0
